client/index.css

```diff
@@ -199,3 +199,112 @@ ul {
 .chat ul li.typing div {
     animation: flash 2s infinite;
 }
+#settings {
+    width: 345px;
+    height: auto;
+    margin: 40px auto 0 auto;
+}
+#settings #icon-username {
+    margin: 18px 0 0 100px;
+    overflow: auto;
+}
+#icon-username h2 {
+    margin-left: 120px;
+}
+#settings .icon {
+    width: 100px;
+    height: 150px;
+    border: 1px solid black;
+    float: left;
+    margin-top: -45px;
```

Comment:
It may help you to [read about floats](https://css-tricks.com/almanac/properties/f/float/) to understand why. The reason is that the floating div.icon is rendering by floating to the left, with additional content present *after* it in the document content appearing to the right of it. Because there is no such content, that space remains empty.